source/transactions/tests/README.rst

```diff
@@ -153,6 +162,7 @@ Then for each element in ``tests``:
    transactions from previous test failures. The command will fail with message
    "operation was interrupted", because it kills its own implicit session. Catch
    the exception and continue.
+##. When testing against a sharded cluster run this command on ALL mongoses.
```

Comment:
Using which MongoClient?  Seems like we'd have to create one MongoClient that's directly connected to each mongos server in the seed list.